ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/fillers/CoreProblemFiller.java

```diff
@@ -150,14 +151,17 @@ private void addImpactOfRangeActionOnCnec(Cnec cnec) {
             }
 
             double sensitivity = linearRaoData.getSensitivity(cnec, rangeAction);
-            double currentSetPoint = rangeAction.getCurrentValue(linearRaoData.getNetwork());
-            // care : might not be robust as getCurrentValue get the current setPoint from a network variant
-            //        we need to be sure that this variant has been properly set
+            if (Math.abs(sensitivity) >= linearRaoParameters.getPstSensitivityThreshold()) {
```

Comment:
You can check first if the rangeAction in a PstRange

Reply:
indeed